scopes/component/merging/merge-cmd.ts

```diff
@@ -196,7 +194,7 @@ ${mergeSnapError.message}
 
   const getFailureOutput = () => {
     if (!failedComponents || !failedComponents.length) return '';
-    const title = '\nthe merge has been skipped on the following component(s)';
+    const title = '\n merge skipped for the following component(s)';
```

Comment:
Please remove the space between the `\n` and the `merge`

Reply:
thanks, done